### PR TITLE
Add zero block into API

### PR DESCRIPTION
**Pull request by uprendis**
- add fake zero genesis block for compatibility with web3 API

### CODE REVIEW

File: gossip/evm_state_reader.go [hunk 1]
```diff
@@ -88,6 +90,13 @@ func (r *EvmStateReader) GetBlock(h common.Hash, n uint64) *evmcore.EvmBlock {
 }
 
 func (r *EvmStateReader) getBlock(h hash.Event, n idx.Block, readTxs bool) *evmcore.EvmBlock {
+	if n == 0 {
+		// fake genesis block for compatibility with web3
+		return evmcore.NewEvmBlock(evmcore.ToEvmHeader(&inter.Block{
+			Time:    inter.Timestamp(1608600000 * time.Second),
```
Comment (hadv) [inline gossip/evm_state_reader.go:96]: nitpick: we can reuse `FakeGenesisTime` from **evmcore**
```suggestion
			Time:    evmcore.FakeGenesisTime,
```

Comment (uprendis) [reply gossip/evm_state_reader.go:96]: It would create a dependency cycle unfortunately

Comment (hadv) [reply gossip/evm_state_reader.go]: I found that we already imported the **evmcore** package for `evmcore.NewEvmBlock`

Comment (uprendis) [reply gossip/evm_state_reader.go]: Oh yes, you're right

**Review by hadv — APPROVED** [review]

**Pull request merged**